tailwind.config.js

```diff
@@ -2,7 +2,16 @@
 export default {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: 'rgb(var(--text-yellow) / <alpha-value>)',
```

Comment:
The ideal case is if you name them the same, I'd suggest this pattern

```ts
primary: 'rgb(var(--primary) / <alpha-value>)', // primary color
foreground: 'rgb(var(--foreground) / <alpha-value>)', // text color
bg: 'rgb(var(--background) / <alpha-value>)', // body background color
"bg-card": 'rgb(var(--background-card) / <alpha-value>)', // card background color
"gradient-from": 'rgb(var(--gradient-from) / <alpha-value>)',
"gradient-to": 'rgb(var(--gradient-to) / <alpha-value>)',
```

This way you can easily switch to dark theme by just setting values for CSS variables